packages/web3-modal/src/components/modals/TransactionsModal/styled.tsx

```diff
@@ -72,7 +72,7 @@ export const TransactionsModalWrapper = styled(ModalContainer).attrs({ modal: 't
   padding: 0.7rem;
 
   a {
-    color: ${(props) => props.theme.modals?.base?.title?.font?.color};
+    color: ${(props) => props.theme.modals?.transactions?.card?.url};
```

Comment:
if you consider my changes [here](https://github.com/PAST3LLE/monorepo/pull/77/commits/7454c1aa18319a4bf41a6e5296dde092e9e5470f#r1507677071) you'll need to change this to `props.theme.modals?.transactions?.card?.text?.url`